docs/workflows/stories-for-multiple-components.md

```diff
@@ -81,7 +82,8 @@ Another option that is more “data”-based is to create a special “story-gen
   paths={[
     'react/list-story-template.js.mdx',
     'react/list-story-template.ts.mdx',
-    'vue/list-story-template.js.mdx'
+    'vue/list-story-template.js.mdx',
+    'angular/list-story-template.ts.mdx'
```

Comment:
```suggestion
    'angular/list-story-template.ts.mdx',
```

Reply:
Fixed with the latest commit. Thank you for checking it :)